src/commands/list.ts

```diff
@@ -1,18 +1,13 @@
-import {drive_v3} from 'googleapis';
-import {
-  drive,
-  loadAPICredentials,
-} from '../auth';
+import { drive_v3 } from 'googleapis';
+import { drive, loadAPICredentials } from '../auth';
 import { URL } from '../urls';
-import {
-  ERROR,
-  LOG,
-  checkIfOnline,
-  logError,
-  spinner,
-} from '../utils';
+import { ERROR, LOG, checkIfOnline, logError, spinner } from '../utils';
 
-const ellipsize = require('ellipsize');
+const ellipsize: (
+  str: string | undefined,
```

Comment:
str?: string,

Reply:
I double check this package code. If str is not a string, an empty string is used. This allows to call the function with null and undefined.